feature/push/src/main/kotlin/de/tum/informatics/www1/artemis/native_app/feature/push/ui/PushNotificationSettingCategoriesListUi.kt

```diff
@@ -144,38 +162,40 @@ private fun PushNotificationSettingEntry(
 //                    onCheckedChanged = { onUpdate(setting.webapp, it, setting.push) }
 //                )
 //            }
+//            }
+        }
 
-            if (setting.push != null) {
-                TextCheckBox(
-                    modifier = Modifier.testTag(TEST_TAG_PUSH_CHECK_BOX),
-                    isChecked = setting.push,
-                    text = stringResource(id = R.string.push_notification_settings_label_push),
-                    onCheckedChanged = { onUpdate(setting.webapp, setting.email, it) }
-                )
-            }
+        if (setting.push != null) {
+            Switch(
+                modifier = Modifier
+                    .scale(0.9f)
```

Comment:
Please introduce a constant for the scale value. This can then also be used in `PushNotificationSettingsUi`